src/branch_view.js

```diff
@@ -53,16 +51,19 @@ function renderHtmlBlob(parent, html, hasApp) {
 	return iframe;
 };
 
-// checks to see if user dismissed Journey previously and whether it should remain dismissed
-function isJourneyDismissed(branchViewData, branch) {
-	// check storage to see dismiss timestamp
-	var dismissTimeStamp = branch._storage.get('hideBanner' + branchViewData["id"], true);
+/**
+ * Checks if a journey should show based on dismiss time
+ * @param       {Object} branch
+ * @return      {boolean}
+ */
+function _areJourneysDismissedGlobally(branch) {
+	var globalDismissEndTimestamp = branch._storage.get('globalJourneysDismiss', true);
 
-	if (dismissTimeStamp === true || dismissTimeStamp > Date.now()) {
+	if (globalDismissEndTimestamp === true || globalDismissEndTimestamp > Date.now()) {
```

Comment:
are there any time zone implications here? Will the `>` be able to handle if the `globalDismissEndTimestamp` is in a different time zone that `Date.now()` returns?

Reply:
Timestamp doesn't have a timezone (well, I guess it does but its always in UTC).

https://github.com/BranchMetrics/web-branch-deep-linking/pull/600/files#diff-e9eb70faeef04133ebfef5782fe6aa2fR572 is where this value gets set. Value is always current timestamp + time the journey has to be dismissed in seconds (https://github.com/BranchMetrics/web-branch-deep-linking/pull/600/files#diff-e9eb70faeef04133ebfef5782fe6aa2fR472)

I'm glad we don't have to worry about timezone here :)